CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Plugin Manager (dd-mm-yyyy)
 
+### 1.0.9 (20-04-2024)
+
+- Making the Plugin Manager look a little cleaner.
```

Comment:
maybe make it a little less vague

Reply:
just say the whole statement, i'll copy paste